src/EditorFeatures/CSharpTest2/Recommendations/RefKeywordRecommenderTests.cs

```diff
@@ -977,5 +977,34 @@ public async Task TestAfterFileScopedNamespace()
 @"namespace N;
 $$");
         }
+
```

Comment:
oops.  part of another fix.  will break out.

Reply:
The using span changes look good. Let me know when the PR is fixed up and I'll give it a final look over.